### PR TITLE
[RFC] Stage 2 for volume device

### PR DESCRIPTION
**Pull request by Trinity2019**
- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)? Yes
- Have you followed the [contributor guidelines](https://github.com/elastic/ecs/blob/main/CONTRIBUTING.md)? Yes
- For proposing substantial changes or additions to the schema, have you reviewed the [RFC process](https://github.com/elastic/ecs/blob/main/rfcs/README.md)? Yes
- If submitting code/script changes, have you verified all tests pass locally using `make test`? N/A
- If submitting schema/fields updates, have you generated new artifacts by running `make` and committed those changes? N/A
- Is your pull request against main? Unless there is a good reason otherwise, we prefer pull requests against main and will backport as needed. Yes
- Have you added an entry to the [CHANGELOG.next.md](https://github.com/elastic/ecs/blob/main/CHANGELOG.next.md)? N/A

### CODE REVIEW

**Comment by Trinity2019** [comment]
@ricardoungureanu @stanek-michal could you review this rfc stage 2 pr?

**Review by ricardoungureanu — APPROVED** [review]

**Review by intxgo — APPROVED** [review]

**Comment by Trinity2019** [comment]
@ebeahan Could you review and merge this PR? Thanks in advance!

File: rfcs/text/0040-volume-device.md [hunk 1]
```diff
@@ -279,9 +360,10 @@ Stage 3: Document resolutions for all existing concerns. Any new concerns should
 
 The following are the people that consulted on the contents of this RFC.
 
- * @Trinity2019    | author
- * @ricardoungureanu| reviewer
- * @stanek-michal  | reviewer
+ * @Trinity2019      | author
```
Comment (ebeahan) [inline rfcs/text/0040-volume-device.md:363]:
```suggestion
 * @Trinity2019      | author, sponsor
```

I overlooked we didn't have a sponsor listed yet. @Trinity2019 would you be willing to list yourself as sponsor?

The sponsor is someone at Elastic who ([source](https://github.com/elastic/ecs/blob/main/rfcs/PROCESS.md#responsibilities-in-this-process)):

* can be the same person as the contributor if they're someone at Elastic that can take ownership of this change
* is involved at least from stage 1 onward if a different person than the contributor
signs off on each stage if a different person than the contributor
* takes or coordinates ownership of the addition in terms of support and maintenance after the RFC process is completed

Comment (Trinity2019) [reply rfcs/text/0040-volume-device.md]: yes, I can be the sponsor. Thanks @ebeahan for reviews :)

**Review by ebeahan — APPROVED** [review]
LGTM. Set advancement date for stage 2.

**Pull request merged**